src/app/onion/learning-object-builder/components/content-upload/app/upload/url-manager/url-manager.component.ts

```diff
@@ -81,19 +86,19 @@ export class UrlManagerComponent implements OnInit, OnDestroy {
     this.add.emit();
     this.triggerSave$.next();
   }
+  
 
   /**
-   * Sets next value of updateUrl by setting title from user input
-   */
-  updateTitle(index: number, title: string) {
-    this.urlUpdated$.next({ index, title, url: undefined });
-  }
-
-  /**
-   * Sets next value of updateUrl by setting url from user input
+   * Function that emits an event when the user enters in both the 
+   * title and url field which also uses objects to store the information entered from the user
```

Comment:
Can we modify this JsDoc comment a little bit? The comment should be less about the nitty gritty details of how the function works and more of a high-level description. Do we need to talk about the function using objects to store information?